data/test-cases/plugins/zoom/general/Settings_slash_commands.md

```diff
@@ -0,0 +1,51 @@
+---
+# (Required) Ensure all values are filled up
+name: "Settings slash command should not entertain any command after `/zoom settings`"
+status: Active
+priority: Normal
+folder: General
+authors: "@arush-vashishtha"
+team_ownership: []
+priority_p1_to_p4: P2 - Core Functions (Do core functions work?)
+
+# (Optional)
+location: null
+component: null
+tags: []
+labels: []
+tested_by_contributor: null
+
+# (Optional) Test type and tools
+cypress: null
+detox: null
+mmctl: null
+playwright: null
+rainforest: []
+manual_test_environments: []
+
+# Do not change
+id: null
+key: null
+created_on: null
+last_updated: null
+case_hashed: null
+steps_hashed: null
+---
+
+**Step 1**
+
+1. Enter the slash command `/zoom` in the chat in any desired channel or DM/GM on MM.
```

Comment:
Try to write enter and run when you do not have any other action in the same step.